src/main/java/com/project/bookstore/model/User.java

```diff
@@ -0,0 +1,27 @@
+package com.project.bookstore.model;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@RequiredArgsConstructor
+@Setter
+@Getter
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "email", nullable = false, unique = true)
+    private String email;
+    @Column(name = "password", nullable = false)
+    private String password;
+    @Column(name = "first_name", nullable = false)
+    private String firstName;
+    @Column(name = "last_name", nullable = false)
+    private String lastName;
+    private String shippingAddress;

```

Comment:
lets stay consistent in naming of our columns
```suggestion
    @Column(name = "shipping_address")
    private String shippingAddress;

```